api/errors.go

```diff
@@ -93,3 +93,8 @@ type Error string
 
 // Error returns the error message.
 func (e Error) Error() string { return string(e) }
+
+// Webhook errors
+const (
+	ErrWebhookAlreadyExists = Error("A webhook for this service already exists")
```

Comment:
With the introduction of a webhook with a type, I believe that we'll want a more generic message such as "A webhook is already associated to this resource"